hack/packet-provision.sh

```diff
@@ -39,7 +39,7 @@ case ${1} in
     ;;
   "destroy")
     terraform destroy -input=false -auto-approve
-    rm /tmp/packet_id_rsa* /tmp/packet_ip 2>/dev/null || :
+    rm -f /tmp/packet_id_rsa* /tmp/packet_ip 2>/dev/null
```

Comment:
Why do we need to hide errors? (i.e., the `2>/dev/null`.

Reply:
No reason.